models/ed/tests/Rcheck_reference.log

```diff
@@ -205,17 +69,101 @@ Extensions’ manual.
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S030W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S060E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S060W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S090E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S090W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S120E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S120W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S150E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S150W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_30S180W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N000E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N030E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N030W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N060E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N060W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N090E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N090W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N120E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N120W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N150E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N150W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60N180W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S000E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S030E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S030W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S060E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S060W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S090E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S090W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S120E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S120W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S150E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S150W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_60S180W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S000E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S030E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S030W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S060E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S060W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S090E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S090W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S120E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S120W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S150E.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S150W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_90S180W.h5’,
+    ‘ed_run_data/EDI/oge2OLD/OGE2_HEADER’, ‘running_ed_from_R.Rmd’
 Package has no Sweave vignette sources and no VignetteBuilder field.
 * checking examples ... OK
-* checking for unstated dependencies in ‘tests’ ... WARNING
-'::' or ':::' import not declared from: ‘devtools’
-* checking tests ... SKIPPED
+* checking for unstated dependencies in ‘tests’ ... OK
+* checking tests ...
+  Running ‘test_ed_integration.R’
+  Running ‘testthat.R’ [12s/12s]
+ [12s/13s] OK
 * checking for non-standard things in the check directory ... OK
 * checking for detritus in the temp directory ... OK
 * DONE
 
-Status: 7 WARNINGs, 4 NOTEs
+Status: 3 WARNINGs, 2 NOTEs
+See
+  ‘/private/var/folders/wr/by_lst2d2fngf67mknmgf4340000gn/T/RtmplbRshU/file7b5043659c65/PEcAn.ED2.Rcheck/00check.log’
```

Comment:
@infotroph should this line be dropped since any future test will put the log in a different temp file? Also what about the timings a few lines up [12s/12s]?

Reply:
@mdietze This line is fine, because the comparison code specifically ignores temp paths. 

Not sure about the timings -- looks like most of our current check logs are run with `--no-tests` and therefore are neither running nor reporting the timing of them, but I no longer remember whether that was a principled decision or an accident. Since the tests are run elsewhere in CI, I don't mind if this log skips them.